### ISSUE TITLE
DEP2 Set the groups order 

### ISSUE THREAD

**Issue by wuxiangjia** [mildpiggy/DEP2#22]
Hello, I have previously commented in another issue. I am still encountering the same problem at the "Set the groups order" step when using "Parse from columns." I will attach a screenshot of my expdesign table. I have used the same sample numbers during the database search in MaxQuant. Thank you for your assistance.
![image](https://github.com/user-attachments/assets/0ca3f633-7724-44d2-9c88-af1a972139e1)
![image](https://github.com/user-attachments/assets/874cb0fd-c0ae-4628-a1f8-629631afb321)

**Comment by mildpiggy** [comment]
Hi. Does it work well when you do not set the order of groups (empty input in this field), 'set the groups order' is not a necessary option?  Moreover, when using 'Parse from columns', the labels of LFQ or expression columns should follow a specific naming convention: condition_repeat. For example, LFQ.intensity.WT_1, LFQ.intensity.WT_2, LFQ.intensity.HZ_1, LFQ.intensity.HZ_2, etc.  You can to modify the columns names in the proteingroups.txt.  
![Uploading image.png…]()

**Comment by wuxiangjia** [comment]
Hello, when I bypass the step of "set the order of groups," the software malfunctions, resulting in the issue shown in Figure 1. Later, after renaming the LFQ or expression column labels in the original file to adhere to specific naming conventions and then using "Parse from columns," the process still fails to run, leading to the situation depicted in Figure 3. The same scenario occurs when I use "expdesign." How can I address this issue? I am very grateful for your guidance and assistance. May I request to contact you directly via email? Thank you once again.
![expdesign](https://github.com/user-attachments/assets/4372a2be-f54f-4a12-8952-3ee92943d7c9)
![Fig1](https://github.com/user-attachments/assets/9ad71a98-113e-4e14-8dc0-bf72e29e33cc)
![Fig2](https://github.com/user-attachments/assets/821719b8-1a11-458b-9815-3bba3d4e949a)
![Fig3](https://github.com/user-attachments/assets/3b6c1d8b-3616-4012-8902-b85e555c984c)

**Comment by mildpiggy** [comment]
Of course. You can contact me directly through this email: feng.zhenhuan@foxmail.com.

**Comment by wuxiangjia** [comment]
Okay, I will contact you via email.

**Comment by wuxiangjia** [comment]
Okay, I will contact you via email.Thank you once again.
